app/components/add-cron-job.js

```diff
@@ -0,0 +1,59 @@
+import Ember from 'ember';
+import config from 'travis/config/environment';
+
+export default Ember.Component.extend({
+  classNames: ['form--cron'],
+  store: Ember.inject.service(),
+
+  reset() {
+    return this.setProperties({
```

Comment:
Is this `return` superfluous? Not worth blocking over, but having a return suggests it’s a promise or something, which it isn’t.